ci-operator/step-registry/baremetalds/e2e/baremetalds-e2e-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: baremetalds-e2e
+  steps:
+    pre:
+    - ref: baremetalds-packet-setup
```

Comment:
Add a step below this:

```
- ref: ipi-install-rbac
```

This will fix the permissions issues on the release image

Reply:
See: https://github.com/openshift/release/pull/7630/files#diff-db2718b3c8c42c85a99504f98c0c3eb3R6